src/main/java/TaskType.java

```diff
@@ -0,0 +1,10 @@
+public enum TaskType {
```

Comment:
This class seems a little redundant to me, I think having too many subclasses may sometimes unnecessarily complicate the code.

Reply:
I see. But I may expand this to an enum of 'Commands' to allow me to switch between different constants in a more reader friendly way. Still thinking about it but thanks for the input!